### DISCUSSION TITLE
Navbar Intergration

### DISCUSSION THREAD

**Discussion by jwaguirr** [freshframework/fresh#373] [Q&A]
Hello! I am fairly new to preact and deno, and am having some basic issues. Can anyone help me with connecting the navigation sidebar to the hamburger menu? Navbars always give me trouble, and am really struggling with the fresh framework. Any Help will be appreciated.

**Comment by rusdyahmad** [comment]
Me too. I just played with Fresh. Maybe you can refer here for a start.

[https://github.com/rusdyahmad/fresh-deno-sample](https://github.com/rusdyahmad/fresh-deno-sample)

**Comment by PhoneDroid** [comment]
@lucacasonato Should close?
- Docs now contains various examples
- Author hasn't been publicly active in a year -> likely no point in asking if they still need help
